### ISSUE TITLE
Can't send E2E messages on Nightly (and it refused to even quit)

### ISSUE THREAD

**Issue by ara4n** [vector-im/riot-web#13347]
https://github.com/matrix-org/riot-web-rageshakes/issues/2685

**Comment by turt2live** [comment]
This looks like a web engine limitation. The crawler is running in your logs, which is hammering the `/messages` endpoint. After a point, Chrome artificially blocks requests to try and prevent the server exploding.

It'll probably catch up normally in enough time.

**Comment by poljar** [comment]
> This looks like a web engine limitation. The crawler is running in your logs, which is hammering the `/messages` endpoint. 

I'm not sure that this is true, there's no indication in the logs that the crawler is doing anything besides waking up and going back to sleep. I'm not saying that it isn't a web engine limitation but it's not because the crawler is hammering the `/messages` endpoint.

**Comment by turt2live** [comment]
That's a lot of logging to be just doing things normally :(

Anyways, in the span of a couple minutes there are more than ~20 opportunities for web requests, which is roughly the same number of requests Chrome starts putting delays in place. Given matrix.org is slow, I'd expect most of them would be still pending.

Easiest way to config/disprove the theory would be having the network console open when trying to send a message. If it sits at pending for a while, it's Chrome.

**Comment by poljar** [comment]
> That's a lot of logging to be just doing things normally :(

Yes, we can tone it down a bit. The committing events one is quite useless anyways. The tick that the crawler is running/sleeping has proven to be useful if it dies for some reason, so we might keep it around a bit longer.
